README.md

```diff
@@ -502,7 +557,7 @@ continue the development of this module.
 
 ## Development
 
-See the [contributing guide](./.github/CONTRIBUTING.md) for details. Additionally, some general guidelines on PR structure can be found [here](https://voxpupuli.org/docs/#reviewing-a-module-pr).
```

Comment:
those are actually two different files

Reply:
Are you referring to the change from `./.github/CONTRIBUTING.md` to `./CONTRIBUTING.md`?